### DISCUSSION TITLE
Able to resolve viem but not viem/accounts

### DISCUSSION THREAD

**Discussion by jerryjfwang** [wevm/viem#403] [Question] [answered]
Hi!

I'm running into a weird issue where I can resolve imports from `viem` but not from `viem/accounts`.

```
// works
import {createWalletClient} from "viem";

// throws "Unable to resolve module viem/accounts"
import {privateKeyToAccount} from "viem/accounts";

// works, but no types
import {privateKeyToAccount} from "../../../node_modules/viem/dist/esm/accounts/index.js"
```

Any idea what could be going on here?

Thanks for the help!

**Comment by jxom** [comment]
What bundler are you using?

**Reply by jerryjfwang** [reply]
Metro bundler. It's a React Native project. I'm using `viem@^0.3.6`

**Reply by jxom** [reply]
Hmm. Metro may not honour package.json#exports. Let me investigate!

**Reply by jxom** [reply, marked as answer]
Looks like it’ll be enabled with a feature flag in 0.72 (hopefully they release it soon)!

https://github.com/facebook/metro/issues/670#issuecomment-1434510835

**Reply by jerryjfwang** [reply]
Amazing. Thanks for the fast response!

**Reply by jxom** [reply]
@jerryjfwang – just released `0.3.31` which should fix this with current versions of Metro.
